### ISSUE TITLE
Ensure builds work on Windows with Android

### ISSUE THREAD

**Issue by asimihsan** [asimihsan/ezmempass#16]
### Problem statement

I've been too wedding to my Mac and I should ensure that running on Android on Windows works.

Out of scope for now is to check this using a fresh Windows virtual machine in e.g. VirtualBox. This would be an excellent way of coming up with high quality instructions, however the intention of this issue is to unblock devs in the short term.

### Acceptance criteria

- Developers can build and run on Windows using an Android emulator.
- Developers continue to be able to build and run on Mac using an Android or iOS emulator.
